spec/components/review_draft/draft/view_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe ReviewDraft::Draft::View do
+  alias_method :component, :page
+
+  before do
+    render_inline(described_class.new(trainee: trainee))
+  end
+
+  context "sections that appear for all trainees regardless of route" do
```

Comment:
This context isn't _quite_ correct if you're doing `let(:trainee) { create(:trainee) }` - our base trainee factory actually always creates an Assessment only trainee. So this is only testing sections that appear for AO trainees, not 'regardless of route'.

Reply:
Makes sense! i've changed to assessment only